restdocs-api-spec-mockmvc/src/main/kotlin/com/epages/restdocs/apispec/MockMvcRestDocumentationWrapper.kt

```diff
@@ -27,7 +27,7 @@ object MockMvcRestDocumentationWrapper : RestDocumentationWrapper() {
             enhanceSnippetsWithResourceSnippet(
                 resourceDetails = resourceDetails,
                 snippetFilter = snippetFilter,
-                snippets = *snippets
+                snippets = snippets
```

Comment:
This an similar changes fixes compilation warnings about the unnecessary usage of the spread operator.